content/applications/websites/livechat/overview/responses.rst

```diff
@@ -1,48 +1,137 @@
+      the website (up to 15).
+
+      .. image:: responses/live-chat-responses-history.png
+         :align: center
+         :alt: View of the results from a /history command in a Live Chat conversation.
+
+   .. tab:: `/lead`
+
+      This command creates a :guilabel:`Lead` in the :guilabel:`CRM` application. After typing
+      `/lead`, create a title for this new lead, then press `Enter`. The transcript of the
+      conversation is added to the :guilabel:`Internal Notes` tab of the lead form. On the
+      :guilabel:`Extra Information` tab of the lead form, the :guilabel:`Source` will be listed as
+      :guilabel:`Livechat`.
+
+      .. image:: responses/live-chat-responses-lead.png
+         :align: center
+         :alt: View of the results from a /lead command in a Live Chat conversation.
+
+   .. tab:: `/leave`
+
+      The `/leave` command lets the *operator* leave the conversation channel.
 
 .. important::
-   | - For *helpdesk tickets*: make sure the application is installed on your database and the
-     option *Live Chat* under :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`
-     is enabled.
-   | - For *leads*: the *CRM* application must be installed on your database.
+   - The `/helpdesk` and `/helpdesk_search` commands can only be used if the :guilabel:`Helpdesk`
+     app has been installed, and :guilabel:`Live Chat` has been activated on a :guilabel:`Helpdesk
+     Team`. To activate :guilabel:`Live Chat`, go to :menuselection:`Helpdesk --> Configuration -->
+     Teams`, and select a team. Scroll to the :guilabel:`Channels` section and check the box
+     labeled :guilabel:`Live Chat`.
+   - The `/lead` command can only be used if the :guilabel:`CRM` app has been installed.
+
+.. tip::
+   To access the :guilabel:`ticket` or :guilabel:`lead` created from the chat, click on the link.
 
-To access the ticket or lead created from the chat, click on the shortcut link.
+  .. image:: responses/live-chat-responses-ticket-link.png
+     :align: center
+     :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat.
 
-.. image:: responses/create_ticket.png
-   :align: center
-   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat
+.. seealso::
+   - :doc:`/applications/sales/crm/acquire_leads`
+   - :doc:`/applications/services/helpdesk/overview/getting_started`
+
+Canned responses
+================
+
+*Canned responses* are customizable inputs where a shortcut stands in for a longer response. An
+operator will enter the :guilabel:`Shortcut`, and it will be replaced by the
+:guilabel:`Substitution`.
+
+Create new canned responses
+---------------------------
+
+To create a new :guilabel:`Canned Response`, go to :menuselection:`Live Chat --> Configuration -->
+Canned Responses --> New`. Choose a :guilabel:`Shortcut`. Enter the :guilabel:`Substitution`
+message. This will be the message that is sent to the recipients. Click :guilabel:`Save`.
```

Comment:
To create a new :guilabel:`Canned Response`, go to :menuselection:`Live Chat application --> Configuration -->
Canned Responses --> New`. 

From here, choose a :guilabel:`Shortcut` by typing into the :guilabel:`Shortcut` field. 

Then, to the right, click into the blank :guilabel:`Substitution` field, and create the custom message that will appear in place of the :guilabel:`Shortcut`. This will be the message that is sent to the recipients. Click :guilabel:`Save` or click away to save the configurations.

(I may even suggest explaining what they see on the Canned Responses page, and then hopping into the create portion)